rmgpy/rmg/input.py

```diff
@@ -225,6 +225,19 @@ def simple_reactor(temperature,
     if not isinstance(temperature, list) and not isinstance(pressure, list) and all(
             [not isinstance(x, list) for x in initialMoleFractions.values()]):
         nSims = 1
+        # normalize mole fractions if applicable
```

Comment:
I think we may want this even if T and P are ranges, since those are independent parameters.

I would add a second if statement here with only `all([not isinstance(x, list) for x in initialMoleFractions.values()])`. Also, the comment could be updated to say `if not using a mole fraction range`.